RELEASE.md

```diff
@@ -0,0 +1,109 @@
+
+* Users can now 
+  [restrict the access of the protocols](https://github.com/triton-inference-server/server/blob/r23.04/docs/customization_guide/inference_protocols.md#limit-endpoint-access-beta) 
+  on a given Triton endpoint.
+
+* Triton now provides a limited support for 
+  [tracing inference requests using OpenTelemetry Trace APIs](https://github.com/triton-inference-server/server/blob/r23.04/docs/user_guide/trace.md#opentelemetry-trace-support).
+
+* Model Analyzer now supports 
+  [BLS Models](https://github.com/triton-inference-server/model_analyzer/blob/r23.04/docs/config_search.md#bls-model-search).
+
+* Refer to the 23.04 column of the 
+  [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) 
+  for container image versions on which the 23.04 inference server container is 
+  based.
+
+## Known Issues
+
+* Tensorflow backend no longer supports TensorFlow version 1.
+
+* [Triton Inferentia guide](https://github.com/triton-inference-server/python_backend/tree/main/inferentia#inferentia-setup) 
+  is out of date. Some users have reported issues with running Triton on AWS 
+  Inferentia instances.
+
+* Some systems which implement `malloc()` may not release memory back to the 
+  operating system right away causing a false memory leak. This can be mitigated 
+  by using a different malloc implementation. Tcmalloc is installed in the 
+  Triton container and can be 
+  [used by specifying the library in `LD_PRELOAD`](https://github.com/triton-inference-server/server/blob/r22.12/docs/user_guide/model_management.md).
+
+* Auto-complete may cause an increase in server start time. To avoid a start 
+  time increase, users can provide the full model configuration and launch the 
+  server with `--disable-auto-complete-config`.
+
+* Auto-complete does not support PyTorch models due to lack of metadata in the 
+  model. It can only verify that the number of inputs and the input names 
+  matches what is specified in the model configuration. There is no model 
+  metadata about the number of outputs and datatypes. Related PyTorch bug: 
+  https://github.com/pytorch/pytorch/issues/38273.
+
+* Triton Client PIP wheels for ARM SBSA are not available from PyPI and pip will 
+  install an incorrect Jetson version of Triton Client library for Arm SBSA.
+
+  The correct client wheel file can be pulled directly from the Arm SBSA SDK 
+  image and manually installed.
+
+* Traced models in PyTorch seem to create overflows when int8 tensor values are 
+  transformed to int32 on the GPU. 
+
+  Refer to https://github.com/pytorch/pytorch/issues/66930 for more information.
+
+* Triton cannot retrieve GPU metrics with MIG-enabled GPU devices (A100 and A30).
+
+* Triton metrics might not work if the host machine is running a separate DCGM 
+  agent on bare-metal or in a container.
```

Comment:
Add a note saying,

"PyTorch backend for Jetson uses the 23.03 release of PyTorch." 

@tanmayv25 to confirm verbage

Reply:
I don't think that we need to add this info here.
I will mention in release notes that we are using proper Torch version
https://docs.nvidia.com/deeplearning/frameworks/install-pytorch-jetson-platform-release-notes/pytorch-jetson-rel.html